tests/storage/storage.go

```diff
@@ -618,10 +618,7 @@ var _ = SIGDescribe("Storage", func() {
 			}, 120)
 
 			// Not a candidate for testing on NFS because the VMI is restarted and NFS PVC can't be re-used
-			It("[Serial][test_id:3138]should start vmi multiple times", func() {
-				// Expansion changes the blockdev output
-				tests.DisableFeatureGate(virtconfig.ExpandDisksGate)
```

Comment:
Can you please explain why this was needed in the first place?

Reply:
I think I changed the calculation for how much to expand a little bit, and got the impression that the last few digits can vary by environment settings.

Reply:
So IIUC disabling the feature gate ensured the size is the same on all environments?
If the last few digits can vary, doesn't it make the test flakie?

The test says `By("Checking that the second disk is present")` which makes me thing that all needed to check here is that the disk exists, and its exact size isn't very relevant. What if instead of specifying the exact size we'd expect a range which is plausible? This way even if size changes by a reasonable amount the test will still be valid

WDYT?

Reply:
Aha, got it. Thanks!